storagenode/storagenodedb/infodb.go

```diff
@@ -360,6 +360,23 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					return nil
 				}),
 			},
+			{
+				Description: "Free Storagenodes from orphaned tmp data",
+				Version:     14,
+				Action: migrate.Func(func(log *zap.Logger, mgdb migrate.DB, tx *sql.Tx) error {
+					// When using inmemory DB, skip deletion process
+					if db.location == "" {
+						return nil
+					}
+
+					err := os.RemoveAll(filepath.Join(filepath.Dir(db.location), "tmp"))
```

Comment:
This location should be passed explicitly in or the migration written in a level higher struct. Once we allow putting pieceinfo to a separate disk, this could be problematic.